### ISSUE TITLE
9.0.0-next.7 gives TypeError: Cannot read property 'before' of undefined

### ISSUE THREAD

**Issue by bbarry** [thymikee/jest-preset-angular#794]
## 🐛 Bug Report

9.0.0-next.6 was working, next.7 gives `TypeError: Cannot read property 'before' of undefined`

## To Reproduce

1. copy test-app-v11 out to a separate directory
2. in new application install next.7 to package.json:

        "jest-preset-angular": "9.0.0-next.7",

3. create file ./src/app/app.component.spec.ts

        import { TestBed, waitForAsync } from "@angular/core/testing";
        import { AppComponent } from "./app.component";

        describe("AppComponent", () => {
          beforeEach(
            waitForAsync(() => {
              TestBed.configureTestingModule({
                declarations: [AppComponent],
              }).compileComponents();
            })
          );

          it("should create the app", () => {
            const fixture = TestBed.createComponent(AppComponent);
            const app = fixture.debugElement.componentInstance;
            expect(app).toBeTruthy();
          });
        });

4. run any of the 4 test scripts

## Expected behavior

pass test

## Error log:

```
# content of error stacktrace :

 FAIL  src/app/app.component.spec.ts
  ● Test suite failed to run

    TypeError: Cannot read property 'before' of undefined

      at NgJestCompiler.getCompiledOutput (node_modules/jest-preset-angular/build/compiler/ng-jest-compiler.js:64:43)
      at ScriptTransformer.transformSource (node_modules/@jest/transform/build/ScriptTransformer.js:439:37)
      at ScriptTransformer._transformAndBuildScript (node_modules/@jest/transform/build/ScriptTransformer.js:549:40)
      at ScriptTransformer.transform (node_modules/@jest/transform/build/ScriptTransformer.js:585:25)

```

## envinfo

```
System:
    OS: Win10

Npm packages:
    jest:  27.0.0-next.2
    jest-preset-angular: 9.0.0-next.7
    typescript: 4.0.5 (actual project: 4.1.3)
```

Seems to be a regression; changing package.json to next.6 and installing makes the test pass.

**Comment by ahnpnl** [comment]
If you install `ts-jest@27.0.0-next.4` it should work because there is a mistake at https://github.com/thymikee/jest-preset-angular/blob/v9.0.0-next.7/package.json#L35 and `next.7` isn’t compatible with `27.0.0-next.5`

**Comment by ahnpnl** [comment]
Closes via #781
